uint8 id      # Landmark identifier (0-20)
int32 x       # x coordinate in pixels
int32 y       # y coordinate in pixels
